pkgs/servers/monitoring/grafana/plugins/yesoreyeram-infinity-datasource/default.nix

```diff
@@ -0,0 +1,18 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin {
+  pname = "yesoreyeram-infinity-datasource";
+  version = "2.6.3";
+  zipHash = {
```

Comment:
I am curious: how did you get the `zipHash`es for the platforms that you aren't using?

Reply:
I downloaded the zip files with `wget`, ran a `sha256sum` on it, then a `nix hash to-sri --type sha256` to get it into SRI format.